pkgs/build-support/php/builders/v2/build-composer-vendor.nix

```diff
@@ -8,14 +8,14 @@ let
   mkComposerVendorOverride =
     finalAttrs:
     {
-      php ? toplevel.php,
-      composer ? toplevel.php.packages.composer,
-      composerLock,
-      vendorHash ? null,
-      composerNoDev ? true,
-      composerNoPlugins ? true,
-      composerNoScripts ? true,
-      composerStrictValidation ? true,
+      php ? finalAttrs.php or toplevel.php,
+      composer ? finalAttrs.php.packages.composer or toplevel.php.packages.composer,
+      composerLock ? finalAttrs.composerLock or null,
+      vendorHash ? finalAttrs.vendorHash or "",
+      composerNoDev ? finalAttrs.composerNoDev or true,
+      composerNoPlugins ? finalAttrs.composerNoPlugins or true,
+      composerNoScripts ? finalAttrs.composerNoScripts or true,
+      composerStrictValidation ? finalAttrs.composerStrictValidation or true,
```

Comment:
`{ <argname> ? finalAttrs.<argname> or <default> }: { }` makes no sense to me, TBH. `{ <argname> ? <default> }` should work.

Reply:
You're right, I fixed it.

Reply:
Yes, that was the idea. Is it OK if I restore it ?

Reply:
I think so.

Reply:
done :)